update

islamshahil opened the pull request:
Take test image using camera

islamshahil committed:
update

Take test image using camera

ionicc commented:
Thanks for the contribution, But try to make these changes:
->The addition of taking a picture from the webcam is great but it shouldn't replace the original test.jpg
Instead make an argparse and ask the user if they want a picture from the webcam or a raw picture from a directory.
->The path to the shape predictor file should be relative as otherwise it will fail in other systems.

Have fun 😄

ionicc reviewed (changes requested):
As mentioned above

islamshahil committed:
Update

Using tkinter, created a message box which allows you to choose between web cam and default picture as your test picture.
